images/macos/scripts/tests/BasicTools.Tests.ps1

```diff
@@ -159,15 +159,15 @@ Describe "Homebrew" {
     }
 }
 
-Describe "Kotlin" {
+Describe "Kotlin" -Skip:($os.IsMonterey) {
     $kotlinPackages = @("kapt", "kotlin", "kotlinc", "kotlinc-jvm", "kotlin-dce-js")
 
     It "<toolName> is available" -TestCases ($kotlinPackages | ForEach-Object { @{ toolName = $_ } }) {
         "$toolName -version" | Should -ReturnZeroExitCode
     }
 }
 
-Describe "sbt" -Skip:($os.IsVentura -or $os.IsSonoma -or $os.IsSequoia) {
+Describe "sbt" -Skip:($os.IsVentura -or $os.IsSonoma -or $os.IsSequoia -or $os.IsMonterey) {
```

Comment:
This skipping all available images. 😄 Not really harmful anyway - might be removed with macOS-12 deprecation process later.